front/components/members/InvitationModal.tsx

```diff
@@ -53,7 +54,8 @@ export function InviteEmailModal({
   const [inviteEmails, setInviteEmails] = useState<string>("");
   const [isSending, setIsSending] = useState(false);
   const [emailError, setEmailError] = useState("");
-  const { mutate } = useSWRConfig();
+  const { mutateInvitations } = useWorkspaceInvitations(owner);
```

Comment:
This will trigger a fetch, I'd suggest to update the props of this component to accept an `onUpdate(resource: "members" | "invitations")` props. Get the mutates from those existing hooks and wire them in the `onUpdate` function.

Reply:
I simplified even more as only one KeyedMutator was needed. Hope I got it right